docs/install/cloud.rst

```diff
@@ -0,0 +1,118 @@
+.. _cloudInstallation:
+
+.. highlight:: console
+
+Cloud installation
+==================
+
+.. _installationAWS:
+
+Amazon Web Services
+-------------------
+Toil is built with an Amazon Web Services provisioner which can be used with
+CGCloud_ to provision instances and clusters in AWS. For more information on
```

Comment:
This sentence is somewhat confusing. Maybe something like `Toil includes a native AWS provisioner that can be used to start autoscaling clusters. For more information on this provisioner, see the Autoscaling section. To provision static, non-autoscaling clusters we recommend using CGCloud`